Standards/scs-0001-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,305 @@
+---
+type: Procedural
+status: Draft
+track: Global
+---
+
+# Introduction
+
+The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
+for a range of cloud infrastructure types.
+It strives for interoperable and sovereign cloud stacks
+which can be deployed and used by a wide range of organizations and individuals.
+Whereever feasible,
+transparency and openness both in respect to the inner workings of the platforms standardised by SCS,
+as well as the SCS organisation itself
+are a paradigm we intend to live.
+
+# Requirements
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+In addition, "FORBIDDEN" is to be interpreted equivalent to "MUST NOT".
+
+# Motivation and Objectives
+
+TBD.
+
+# Sovereign Cloud Standard documents
+
+One of the main products of the SCS organisation are Sovereign Cloud Standard documents.
+
+## Types of documents
+
+### Procedural
+
+A procedural SCS document describes a process, a policy or a guideline
+to which the SCS community adheres.
+
+### Standard
+
+A standard SCS document describes a technical standard for SCS compliant clouds.
+Note that it may not be necessary for all clouds to implement all standards.
+
+### Decision Record
+
+Sometimes during the development of the SCS standard,
+a complex technical decision needs to be taken,
+which does not directly result in a new standard.
+
+To keep a record of such decisions for future reference,
+they were formerly documented in a format called
+"Architecture Decision Records" (ADRs).
+
+The SCS document format supersedes and integrates such ADRs
+as documents of type `Decision Record`.
+
+## Document format
+
+The SCS documents are provided in GitHub flavored markdown.
+Each document is assigned a unique number.
+To disambiguate with other organisations using similar schemes
+(such as XEPs, PEPs or IETF RFCs),
+the numbers are prefixed with `SCS-`.
+
+To allow a concept to evolve while allowing breaking changes,
+each SCS document is associated with a major version number.
+This major version number is a positive number
+and the numbering starts at one for each document.
+
+In order to make organisation of the SCS documents easier,
+each document also has a slugified title.
+The slugified title MUST NOT be changed after the acceptance of the document into the repository,
+as it is part of its canonical URL.
+It MUST consist only of lower-case ASCII letters, numbers and hyphens.
+It MUST NOT start with a hyphen and SHOULD start with a lower-case letter.
+It SHOULD NOT contain more than one subsequent hyphen and SHOULD start with a lower-case letter.
```

Comment:
`SHOULD start with a lower-case letter` is a duplicate of the second part of the prior sentence here, isn't it?

Reply:
Fixed, thanks!